testdata/p4_16_errors_outputs/issue2413.p4-stderr

```diff
@@ -0,0 +1,6 @@
+issue2413.p4(23): error: MethodCallStatement: No argument supplied for b
```

Comment:
is the `MethodCallStatement` here expected? I'm surprised it doesn't use the action name.

Reply:
I have improved this error message